_rules/aria-hidden-no-focusable-content-6cfa84.md

```diff
@@ -32,11 +32,11 @@ This rule applies to any element with an `aria-hidden` [attribute value][] of `t
 
 ## Expectation
 
-None of the target elements are [focusable][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are [focusable][].
+None of the target elements are part of the [sequential focus navigation][], nor do they have [descendants](https://dom.spec.whatwg.org/#concept-tree-descendant) in the [flat tree](https://drafts.csswg.org/css-scoping/#flat-tree) that are part of the [sequential focus navigation][], unless, upon receiving focus, focus is moved to an element that does not fail this rule.
```

Comment:
We might be able to rephrase like
```suggestion
None of the target elements or their [descendants][] in the [flat tree][] are part of the [sequential focus navigation][], unless, upon receiving focus, focus is moved to an element that does not fail this rule.
```
or maybe
```suggestion
None of the target elements has [inclusive descendants][] in the [flat tree][] that are part of the [sequential focus navigation][],  upon receiving focus, focus is moved to an element that does not fail this rule.
```
which does keep the text more "focused" on the targets.